app/pages/sections/info/DestinationTypePage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections.info
+
+import models.sections.info.DestinationType
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object DestinationTypePage extends QuestionPage[DestinationType] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
Should have `info` as part of the path:
```suggestion
  override def path: JsPath = JsPath \ "info" \ toString
```